logicalplan/plan.go

```diff
@@ -136,10 +135,10 @@ func Traverse(expr *parser.Expr, transform func(*parser.Expr)) {
 
 func TraverseBottomUp(parent *parser.Expr, current *parser.Expr, transform func(parent *parser.Expr, node *parser.Expr) bool) bool {
 	switch node := (*current).(type) {
-	case *parser.StringLiteral:
-		return false
-	case *parser.NumberLiteral:
-		return false
+	case *parser.StringLiteral, *StringLiteral:
```

Comment:
Maybe not something for now, but I think long term all logical nodes should be value objects instead of pointers.

Reply:
I agree! Lets do that once we have all logical plan nodes in place!